frontend/src/app/components/team/team.component.ts

```diff
@@ -16,16 +19,22 @@ import { DialogService } from '../../services/dialog.service';
 export class TeamComponent implements OnInit {
   @Input({ required: true })
   public overviewEntity!: OverviewEntity;
+  protected addIconSrc: BehaviorSubject<string> = new BehaviorSubject('/assets/icons/new-icon.svg');
 
   constructor(
     private dialogService: DialogService,
     private refreshDataService: RefreshDataService,
-  ) {}
-
-  trackByObjectiveId: TrackByFunction<ObjectiveMin> = (index, objective) => objective.id;
+    private configService: ConfigService,
+  ) {
+    this.configService.config$.subscribe((config: ClientConfig) => {
+      this.addIconSrc.next(config.customStyles['okr-add-objective-icon'] ?? '/assets/icons/new-icon.svg');
```

Comment:
```suggestion
      this.addIconSrc.next(config.customStyles['okr-add-objective-icon']);
```
I think this  is redundant, as the `BehaviorSubject` already has the same default value :thinking: 
I'd rather have a check if the `customStyles[...]` value is `null` and only changing it if it isn't.